data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+				`	(Do not mention the Hai.)`
+					goto interesting
+			`	"I have actually been there," you say, "It leads to a nearby part of space inhabited by the Hai, an advanced alien civilization of squirrel-like creatures."`
+			`	"Really? Are they friendly?"`
+			choice
+				`	"Yes, they even allow humans to settle there or buy their technology."`
+					goto doctor
+				`	"Yes, though I'm worried they can conquer all of human space if they wish."`
+					goto doctor
+			label interesting
+			`	"Sounds interesting," you say, "Could it lead to some aliens, or perhaps to some worlds waiting to be colonized?"`
+			`	"Not sure. It is located in a lawless barren region of space, however, so I wouldn't be surprised if there are pirates or even Alphas on the other side."`
+			label doctor
+			`	Just as the conversation hangs for a moment, the door opens and a doctor in protective clothing steps in.`
+			`	"Who are you? And why are you in this room?", the doctor inquires, in a soft but authoritative voice.`
+			choice
+				`	"I am <first> <last>, here to see this friend of mine."`
+				`	"I was.. just strolling around."`
+			`	"Nevermind what you were doing. I have to warn you that the patient here has a disease we do not know much about yet. It could potentially be some deadly, contagious disease. Now, please make haste and remove yourself from the room, so as to not risk catching it yourself."`
+			`	The doctor accompanies his words with a lot of vigorous movements, and you find yourself outside the room before realizing it. As the door swings shut, you hear something of a chuckle, probably from James.`
+				goto end
+			label wait
+			`	Soon enough, a doctor passes by, heading for the room.`
+			`	"Good day! How can I help you?", he inquires.`
+			`	"I just wanted to check on a friend of mine, James, who I heard is in this room. May I see him?"`
+			`	"Well, he is infected with some disease. We are not sure what it is exactly and if it is contagious or not. So I would recommend not seeing him in person, at least for the moment," he says just before entering James's room.`
+			label end
+			choice
+				`	(Wait outside the room.)`
+			`	Around half an hour later, the doctor finally exists the room.`
+			`	"I see you probably want to help," he says, "I need someone to go to the spaceport and find a captain willing to bring some newly-discovered medication from <destination> as fast as possible. Can you do that?"`
+			choice
+				`	"Actually I am a spaceship captain. I could do that."`
+				`	"Yes, I can find a capable spaceship captain."`
+					goto `second chance`
+				`	"I'm to busy to run errands. Ask someone else."`
+					decline
+			`	"You are? Superb! Come back as soon as possible. We wouldn't want to risk your friend's life for a minute delay."`
+				accept
+			label `second chance`
+			`	On the way to the spaceport you realize that you do not have to find another spaceship captain to bring medicine from <planet>. You could do it yourself!`
```

Comment:
Why even give the option to start looking then if the player is going to just "realize" they can do it on their own?